ui/apps/everest/src/pages/databases/DbClusterView.tsx

```diff
@@ -127,7 +127,7 @@ export const DbClusterView = () => {
             alignItems="center"
             gap={1}
           >
-            <DbTypeIconProvider dbType={row.original?.dbType} />
+            {beautifyDbTypeName(dbEngineToDbType(row.original?.dbType))}{' '}
```

Comment:
i'm nit sure in this place, i mean {' '}, probably we can do it better, but i don't know how, so let's up to upstream